string sender
string shortestPath
bool enableFlag
string xCoordinates
string yCoordinates
